csharp/src/elements/Geometry/Polygon.cs

```diff
@@ -69,6 +69,47 @@ public Vector3 Centroid
         /// <exception cref="System.ArgumentException">Thrown when coincident vertices are provided.</exception>
         public Polygon(IList<Vector3> vertices) : base(vertices){}
 
+        /// <summary>
+        /// Tests if the supplied 2D point is within the perimeter of this Polygon.
+        /// </summary>
+        /// <param name="point">The 2D Vector3 point to compare.</param>
+        /// <returns>
+        /// Returns true if the supplied Vector3 point is inside this Polygon.
```

Comment:
Can we extend this comment to say "Returns true if the supplied Vector3 is inside this Polygon, and false if the supplied Vector3 is outside this Polygon or coincident with it."? The edge condition is when the point is on the polygon. The comment should clarify what happens in that situation.

Reply:
Absolutely.

Reply:
Done.